src/components/presentational/Chat/Chat.css

```diff
@@ -0,0 +1,75 @@
+.mdhui-chat {
+    background: var(--mdhui-background-color-0);
+    font-size: 14px;
+}
+
+.mdhui-chat .mdhui-chat-log {
+    overflow-y: auto;
+    overflow-x: hidden;
+    min-height: 20vh;
+}
+
+.mdhui-chat p,
+.mdhui-chat ul {
+    margin: .25rem 0;
+    word-break: break-word;
+    color: var(--mdhui-text-color-1);
+}
+
+.mdhui-chat input,
+.mdhui-chat button {
+    min-height: var(--mdhui-touch);
+    min-width: var(--mdhui-touch);
+    border: 0;
+    border-radius: var(--mdhui-card-border-radius);
+}
+
+.mdhui-chat .send-button {
+    position: absolute;
+    right: 4px;
+    cursor: pointer;
+    text-align: center;
+}
+
+.mdhui-chat input {
+    flex: 1;
+    font-size: 1em;
+    color: var(--mdhui-text-color-1);
+    padding: var(--mdhui-padding-md) var(--mdhui-touch) var(--mdhui-padding-md) var(--mdhui-padding-md);
+    box-sizing: border-box;
+    outline-offset: -1px;
+}
+
+.mdhui-chat .input-group {
+    position: relative;
+    display: flex;
```

Comment:
Confused, this is display:flex but the button is positioned absolute?  Seems like you should just use flex to lay out the input / button?

Reply:
If I don't do it like this the outline won't cover the send icon.

<img src="https://github.com/user-attachments/assets/3f9984fd-3bc4-45f2-8765-1f7a2cf33111" width="1000" />